packages/app/src/components/home/HomePage.tsx

```diff
@@ -14,101 +14,78 @@
  * limitations under the License.
  */
 
+import { Page, Content } from '@backstage/core-components';
 import {
-  ClockConfig,
-  CustomHomepageGrid,
-  HeaderWorldClock,
   HomePageCompanyLogo,
-  HomePageRandomJoke,
+  TemplateBackstageLogo,
   HomePageStarredEntities,
   HomePageToolkit,
   HomePageTopVisited,
   HomePageRecentlyVisited,
-  WelcomeTitle,
 } from '@backstage/plugin-home';
-import { Content, Header, Page } from '@backstage/core-components';
 import { HomePageSearchBar } from '@backstage/plugin-search';
-import HomeIcon from '@material-ui/icons/Home';
+import { SearchContextProvider } from '@backstage/plugin-search-react';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
 
-const clockConfigs: ClockConfig[] = [
-  {
-    label: 'NYC',
-    timeZone: 'America/New_York',
-  },
-  {
-    label: 'UTC',
-    timeZone: 'UTC',
-  },
-  {
-    label: 'STO',
-    timeZone: 'Europe/Stockholm',
-  },
-  {
-    label: 'TYO',
-    timeZone: 'Asia/Tokyo',
-  },
-];
+import { tools, useLogoStyles } from './shared';
 
-const timeFormat: Intl.DateTimeFormatOptions = {
-  hour: '2-digit',
-  minute: '2-digit',
-  hour12: false,
-};
-
-const defaultConfig = [
-  {
-    component: 'CompanyLogo',
-    x: 0,
-    y: 0,
-    width: 12,
-    height: 1,
-    movable: false,
-    resizable: false,
-    deletable: false,
-  },
-  {
-    component: 'WelcomeTitle',
-    x: 0,
-    y: 1,
-    width: 12,
-    height: 1,
+const useStyles = makeStyles(theme => ({
+  searchBarInput: {
+    maxWidth: '60vw',
+    margin: 'auto',
+    backgroundColor: theme.palette.background.paper,
+    borderRadius: '50px',
+    boxShadow: theme.shadows[1],
   },
-  {
-    component: 'HomePageSearchBar',
-    x: 0,
-    y: 2,
-    width: 12,
-    height: 2,
+  searchBarOutline: {
+    borderStyle: 'none',
   },
-];
+}));
 
-export const homePage = (
-  <Page themeId="home">
-    <Header title={<WelcomeTitle />} pageTitleOverride="Home">
```

Comment:
Why are we dropping the Welcome Title and the World Clocks?

Reply:
Good catch, I removed them because I redid the homepage based on the demo site one which is more updated and the demo one did not have those two but I can add them again, I will do it today.

Reply:
Done, I added them to the homepage but left them out of the customizable homepage so we have both options in the example. 
![image](https://github.com/user-attachments/assets/6615c639-59ca-4e1c-9e0b-dd6c98b34ef4)